core/v0/api/core.yaml

```diff
@@ -1872,14 +1873,14 @@ components:
         tags:
           $ref: '#/components/schemas/Tags'
     Gps:
-      description: Describes a gps coordinate
+      description: Describes a GPS coordinates which is a unique identifier of a precise geographic location on the earth.<br>
       type: string
       pattern: '^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?),\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$'
     Image:
       description: 'Image of an object. <br/><br/> A url based image will look like <br/><br/>```uri:http://path/to/image``` <br/><br/> An image can also be sent as a data string. For example : <br/><br/> ```data:js87y34ilhriuho84r3i4```'
       type: string
     Intent:
-      description: Intent of a user. Used for searching for services
+      description: Intent of a user. Used for searching for products or services.<br>This has properties like descriptor,provider,fulfillment,payment,category,offer,item,tags<br>This is typically used by the BAP to send the purpose of the user's search to the BPP. This will be used by the BPP to find products or services it offers that may match the user's intent.
```

Comment:
Maybe couple of sentences are useful for developers to understand that the intent schema is layered to accommodate a journey intent in mobility domain is represented differently from a lab booking intent in health domain.